packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -557,7 +558,11 @@ export default [
                     <label>
                       <strong>{_('backupName')}</strong>
                     </label>
-                    <Input onChange={effects.setName} value={state.name} />
+                    <Input
+                      className='form-control'
+                      onChange={effects.setName}
+                      value={state.name}
+                    />
```

Comment:
Why does this change fix the problem?

Reply:
It's not the same `Input`. I use now `debounce-input-decorator` to debounce the react input.

Reply:
How is it related to the initial problem?

Reply:
The cause of the cursor jumping is when the state change the input is re-render by react that's why we need to debounce the input.

Reply:
No: [example](https://codesandbox.io/s/wyn7wv4o98).

Reply:
The issue is caused by a controlled input.

Reply:
Update `@julien-f/freactal` to 0.1.1.